include/rocksdb/metadata.h

```diff
@@ -117,6 +119,8 @@ struct SstFileMetaData {
   // oldest SST file that is the compaction ancester of this file.
   // The timestamp is provided Env::GetCurrentTime().
   // 0 if the information is not available.
+  //
+  // Note: for TTL blob files, it contains the start of the expiration range.
   uint64_t oldest_ancester_time;
```

Comment:
Before this change, this field would generally hold a timestamp in the past, but now for blob files it will often be in the future?

Just an oddity. An alternative might not be better.

Reply:
Agreed. The field does serve a similar purpose though for both SSTs and TTL blob files.